build_library/grub_install.sh

```diff
@@ -140,6 +140,10 @@ info "Generating ${GRUB_DIR}/load.cfg"
 ESP_FSID=$(sudo grub-probe -t fs_uuid -d "${LOOP_DEV}p1")
 sudo_clobber "${ESP_DIR}/${GRUB_DIR}/load.cfg" <<EOF
 search.fs_uuid ${ESP_FSID} root \$root
```

Comment:
I don't know, perhaps by querying UEFI for the boot disk? I don't know if there is an existing mechanism to translate what UEFI is using at the ESP to grub disk name bit if there is that'd probably be better than searching for a fs uuid like this.